src/e2e/java/uk/gov/hmcts/reform/sscs/functional/handlers/requesthearingrecording/RequestHearingRecordingAboutToSubmitHandlerTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.sscs.functional.handlers.requesthearingrecording;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import org.apache.http.HttpStatus;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.functional.handlers.BaseHandler;
+
+@RunWith(SpringRunner.class)
+@TestPropertySource(locations = "classpath:config/application_e2e.properties")
+@SpringBootTest
+public class RequestHearingRecordingAboutToSubmitHandlerTest extends BaseHandler {
```

Comment:
Minor comment but do you mind adding the word Functional before the word 'Test' as we've done for some of the other functional, otherwise there are 2 tests with same name which is abit annoying when trying to find the correct one.